velox/exec/tests/MergeJoinTest.cpp

```diff
@@ -592,6 +592,38 @@ TEST_F(MergeJoinTest, semiJoin) {
       core::JoinType::kRightSemiFilter);
 }
 
+TEST_F(MergeJoinTest, rightJoin) {
+  auto left = makeRowVector(
+      {"t0"},
+      {makeNullableFlatVector<int64_t>(
+          {1, 2, std::nullopt, 5, 6, std::nullopt})});
+
+  auto right = makeRowVector(
```

Comment:
could you also extend join fuzzer to ensure it now can generate plans containing right merge joins?

Reply:
@pedroerp Already enable the right join [here](https://github.com/facebookincubator/velox/pull/10148/files#diff-09674351a7aef3cdea57735e87fdbabfb754300c5c78cc35ea83d75811d41662R864).